eZ/Publish/Core/Persistence/Legacy/Content/Type/ContentUpdater/Action/RemoveField.php

```diff
@@ -94,6 +94,13 @@ function ($versionNo) use ($contentId) {
             );
         }
 
+        // Delete from relations storage
+        if (
+            $this->fieldDefinition->fieldType === 'ezobjectrelation'
+            || $this->fieldDefinition->fieldType === 'ezobjectrelationlist'
```

Comment:
well... moving those literals to some consts in Relation and RelationList Type classes wouldn't hurt. Here for readability you could refactor it as a private/local method: `isRelationFieldType(FieldDefinition $fieldDefinition): bool`